### PR TITLE
[TMUX] Add Tmux spec

### PR DESCRIPTION
**Pull request by QuiiBz**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
No spec for `tmux` cli

**What is the new behavior (if this is a feature change)?**
Add a spec for `tmux`

**Additional info:**

### CODE REVIEW

**Comment by cstrnt** [comment]
@QuiiBz is this still in Draft?

**Comment by QuiiBz** [comment]
Yes, I didn't push yet my local commits. The `tmux `spec has tons of subcommand so this will take a little longer than expected.

**Comment by cstrnt** [comment]
No worries, take your time, Tom 😊

**Comment by QuiiBz** [comment]
Ready for a review!

**Review by cstrnt — CHANGES_REQUESTED** [review]
What a big PR!

**Comment by QuiiBz** [comment]
Could we get this merged? GitHub marked few comments in the above review as not outdated, but the changes have been made underneath.

File: dev/tmux.ts [hunk 1]
```diff
@@ -0,0 +1,2133 @@
...
+        {
+          name: "-N",
+          description: "Attach a note to the key",
+          args: {
+            name: "note",
+          },
+        },
+        {
+          name: "-T",
+          description: "THe key table to use",
```
Comment (cstrnt) [inline dev/tmux.ts:125]:
```suggestion
          description: "The key table to use",
```

File: dev/tmux.ts [hunk 2]
```diff
@@ -0,0 +1,2133 @@
...
+        {
+          name: "-t",
+          description: "The target window",
+          args: windowsArg,
+        },
+      ],
+    },
+    {
+      name: ["wait", "wait-for"],
+      description: "Wait for an event or trigger it",
```
Comment (cstrnt) [inline dev/tmux.ts:2130]: This should take an argument, right?

File: dev/tmux.ts [hunk 3]
```diff
@@ -0,0 +1,2133 @@
...
+          name: "-T",
+          description: "Set the client's key table",
+          args: {
+            name: "key-table",
+          },
+        },
+      ],
+    },
+    {
+      name: ["unbind", "unbind-key"],
```
Comment (cstrnt) [inline dev/tmux.ts:2110]: This should take an argument as well

**Review by cstrnt — CHANGES_REQUESTED** [review]

**Review by cstrnt — APPROVED** [review]
LGTM

**Pull request merged**